letsencrypt/client/auth_handler.py

```diff
@@ -387,13 +395,14 @@ def _find_dumb_path(challs, preferences):
     assert len(preferences) == len(set(preferences))
 
     path = []
-    satisfied = set()
+    # This cannot be a set() because POP challenge is not currently hashable
```

Comment:
What do you mean exactly (error traceback, etc.)? I'd like to fix it. And if you don't care to wait for the fix, could you mark this line as "TODO" so it's easier to spot?